lib/Core/TerriaError.ts

```diff
@@ -155,6 +174,15 @@ export default class TerriaError {
     }
   }
 
+  get severity() {
+    const nestedSeverity = this.flatten().map(error => error._severity);
+    if (nestedSeverity.includes(TerriaErrorSeverity.Severe))
+      return TerriaErrorSeverity.Severe;
+    if (nestedSeverity.includes(TerriaErrorSeverity.Error))
+      return TerriaErrorSeverity.Error;
+    return TerriaErrorSeverity.Warning;
```

Comment:
I created a separate `nestedSeverity` getter.

https://github.com/TerriaJS/terriajs/blob/6babb3fe18fcd8b9d00fe78d52c0487acc0c1ba8/lib/Core/TerriaError.ts#L173-L201

Reply:
I ended up removing `nestedSeverity` - now error severity will default to `Error`.

When an error is cloned - the severity will be preserved - it can also be overridden